flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/utils/ParserUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.client.cli.utils;
+
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.internal.TableEnvironmentInternal;
+import org.apache.flink.table.client.gateway.SqlExecutionException;
+import org.apache.flink.table.operations.Operation;
+
+import java.util.List;
+
+/**
+ * An utility class that provides abilities to parse sql statements.
+ */
+public class ParserUtils {
+
+	private static final TableEnvironment tableEnv = TableEnvironment.create(EnvironmentSettings.newInstance().build());
+
+	static {
+		tableEnv.executeSql("create table MyTable (a int, b bigint, c varchar(32)) " +
+				"with ('connector' = 'filesystem', 'path' = '/non')");
+		tableEnv.executeSql("create table MyOtherTable (a int, b bigint) " +
+				"with ('connector' = 'filesystem', 'path' = '/non')");
```

Comment:
It's a `ParserUtils` but why we register table there ?

Reply:
prepare some tables for testing. now `Parser#parse` contains validation logic, we should make sure the table in a statement exists. I can add some comments here

Reply:
I think the tool class should be stateless, or we make a common data structure there with pre-registered tables there only for testing.